net/databag/Makefile

```
@@ -0,0 +1,69 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=databag
+PKG_VERSION:=1.1.11
+
+PKG_SOURCE:=$(PKG_NAME)-v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL_FILE:=server.tar.gz
+PKG_SOURCE_URL:=https://github.com/balzack/databag/releases/download/v$(PKG_VERSION)/
+PKG_HASH:=b4f9b46ffc9810156c6b8e50216ee4b703b5967a3a88acd982e0ce9ee42098e4
+
+PKG_MAINTAINER:=Roland Osborne <roland.osborne@gmail.com>
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/databag-openwrt-$(PKG_VERSION)
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+PKG_BUILD_FLAGS:=no-mips16
+
+UNPACK_CMD=tar xf "$(DL_DIR)/$(PKG_SOURCE)" -C $(1)
```

```suggestion
TAR_CMD=$(HOST_TAR) -C $(1) $(TAR_OPTIONS)
```

move it below `include ../../lang/golang/golang-package.mk`